src/event.rs

```diff
@@ -74,9 +74,94 @@ pub enum Event<'a, T: 'static> {
+    ///
+    /// After being `Suspended` on Android applications must drop all render surfaces before
+    /// the event callback completes, which may be re-created when the application is next `Resumed`.
+    ///
+    /// [Activity lifecycle]: https://developer.android.com/guide/components/activities/activity-lifecycle
+    /// [`VkSurfaceKHR`]: https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkSurfaceKHR.html
+    /// [`wgpu::Surface`]: https://docs.rs/wgpu/latest/wgpu/struct.Surface.html
+    ///
+    /// ## iOS
+    ///
+    /// On iOS the `Suspended` event is currently emitted in response to an
+    /// [`applicationWillResignActive`] callback which means that the application is
+    /// about to transition from the active to inactive state (according to the
+    /// [iOS application lifecycle])
+    ///
+    /// [`applicationWillResignActive`]: https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622950-applicationwillresignactive
+    /// [iOS application lifecycle]: https://developer.apple.com/documentation/uikit/app_and_environment/managing_your_app_s_life_cycle
     Suspended,
 
     /// Emitted when the application has been resumed.
+    ///
+    /// For consistency all platforms emit a `Resumed` event even if they don't themselves have a
+    /// formal suspend/resume lifecycle. For systems without a standard suspend/resume lifecycle
+    /// the `Resumed` event is is always emitted after the `NewEvents(StartCause::Init)` event.
+    ///
+    /// # Portability
+    ///
+    /// It's recommended that applications should only initialized their graphics context and create
+    /// a window after they have received their first `Resumed` event. Some systems
+    /// (specifically Android) won't allow applications to create a render surface until they are
+    /// resumed.
+    ///
+    /// In case of corner cases with how platform-specific events are mapped to Winit `Suspended`
+    /// and `Resumed` events it's recommended that applications be able to gracefully handle redundant
+    /// (i.e. back to back) `Suspended` or `Resumed` events.
+    ///
+    /// Also see [`Suspended`][Self::Suspended] notes.
+    ///
+    /// ## Android
+    ///
+    /// On Android specifically, the `Suspended` event relates to two things:
+    /// 1. The application has resumed (according to the [Activity lifecycle])
+    /// 2. And the application has an associated SurfaceView (which is expected to closely correlate
+    /// with being resumed)
+    ///
+    /// Applications that need to run on Android must wait until they have been `Resumed`
+    /// before they will be able to create a render surface (such as an `EGLSurface`,
+    /// [`VkSurfaceKHR`] or [`wgpu::Surface`]) which depend on having a
+    /// SurfaceView. Applications must also assume that if they are `Suspended` then their
+    /// render surfaces are invalid and should be dropped. (Also see [`Suspended`][Self::Suspended] notes)
```

Comment:
```suggestion
    /// render surfaces are invalid and should be dropped.
    ///
    /// Also see [`Suspended`][Self::Suspended] notes.
```

To match what you wrote there.